Formula/a/aws-sam-cli.rb

```diff
@@ -3,8 +3,8 @@ class AwsSamCli < Formula
 
   desc "CLI tool to build, test, debug, and deploy Serverless applications using AWS SAM"
   homepage "https://aws.amazon.com/serverless/sam/"
-  url "https://files.pythonhosted.org/packages/7a/6c/b028e3128229318d75b328458395e99be2281ad06b71e97c86d86664334d/aws-sam-cli-1.98.0.tar.gz"
-  sha256 "4fb4f2c24ceb7fb86d5db25c714ccbc5f40fa7a2a4f664202396653ce3a6397e"
+  url "https://files.pythonhosted.org/packages/e3/b5/cc0f3590e3b3891038e6a3e2158a7bcffecdba195896ec632e1447fad220/aws-sam-cli-1.99.0.tar.gz"
+  sha256 "a3adebd8368145a1c19fd2d6ed6ed54de1d5d7629a9982a5c35031e6762b4e91"
   license "Apache-2.0"
   revision 1
```

Comment:
![Uploading image.png…]()

Reply:
relates to https://github.com/Homebrew/homebrew-core/pull/151500

---

you need to update your base branch before next PR.